sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -32,6 +32,7 @@
   private Map<String, String> tags = new HashMap<>();
   private Map<String, Object> extra = new HashMap<>();
   private Map<String, Object> unknown;
+  // TODO modules is missing?
```

Comment:
@bruno-garcia do we need that?

Reply:
Last I looked it wasn't possible to find at runtime the loaded modules(packages) in the process.
Is it possible in Android ?
Otherwise we can just delete the comment
```suggestion
```